src/kits/tracker/FilePanelPriv.cpp

```diff
@@ -1638,21 +1604,11 @@ TFilePanel::SwitchDirMenuTo(const entry_ref* ref)
 }
 
 
-void
-TFilePanel::WindowActivated(bool active)
-{
-	// force focus to update properly
-	fBackView->Invalidate();
-	_inherited::WindowActivated(active);
-}
-
-
 //	#pragma mark -
 
 
-BFilePanelPoseView::BFilePanelPoseView(Model* model, BRect frame,
-	uint32 resizeMask)
-	: BPoseView(model, frame, kListMode, resizeMask),
+BFilePanelPoseView::BFilePanelPoseView(Model* model)
+	: BPoseView(model, kListMode),
```

Comment:
Actually, these changes are probably going to get reverted. But I'll fix the style anyway.